src/parser/jobs/dnc/changelog.tsx

```diff
@@ -8,43 +8,48 @@ export const changelog = [
 	// 	contrubutors: [CONTRIBUTORS.YOU],
 	// },
 	{
-		date: new Date('2021-11-04'),
-		Changes: () => <>Initial data scaffolding, updated proc handling, and add AoE vs. single-target checks for revised proc actions.</>,
+		date: new Date('2023-09-04'),
+		Changes: () => <>Fix a bug related to the Technical Finish status application changes from 6.4 that occasionally caused incorrect 'missed' Finish reports.</>,
 		contributors: [CONTRIBUTORS.AKAIRYU],
 	},
 	{
-		date: new Date('2021-11-27'),
-		Changes: () => <>Removed combo drop leniency around back-to-back Standard/Technical Steps due to lengthened combo timeout.</>,
+		date: new Date('2023-08-19'),
+		Changes: () => <>Esprit and Feather Gauges now show the actual values for logs the Dancer uploaded themselves! Overcap is still partly simulated though.</>,
 		contributors: [CONTRIBUTORS.AKAIRYU],
```

Comment:
Update changelog entry.  Make sure we don't have any other unintended changes happening here and it's just GH being weird with the change displayed.

Reply:
Yeah, fixing the changelogs on all these stale PRs is definitely on the list to do before we approve them. Should be good here with the latest commit.